.github/workflows/_test_pax_rosetta.yaml

```diff
@@ -36,7 +41,7 @@ jobs:
     runs-on: ubuntu-22.04
 
     env:
-      BADGE_FILENAME_FULL: rosetta-pax-single-process-multi-device-te
+      BADGE_FILENAME_PREFIX: rosetta-pax-single-process-multi-device-te
```

Comment:
Do we miss the `badge-` prefix here?

Reply:
Oops. I missed that one. Good catch. I'll fix it